testprojects/src/scala/org/pantsbuild/testproject/javadepsonscalatransitive/ScalaInDifferentPackage.scala

```diff
@@ -0,0 +1,6 @@
+// Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
```suggestion
// Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```
We should add another pre-commit hook for this, I'll make an issue.